www_src/lib/render.jsx

```diff
@@ -1,9 +1,12 @@
 var React = require('react');
+
 var Spindicator = require('../components/spindicator/spindicator.jsx');
 var ModalConfirm = require('../components/modal-confirm/modal-confirm.jsx');
 var ModalSwitch = require('../components/modal-switch/modal-switch.jsx');
 var Snackbar = require('../components/snackbar/snackbar.jsx');
 
+var platform = require('./platform');
```

Comment:
This is one of the places where I migrated to using the polyfill. Everything should migrate to using the `platform` module eventually but in the interest of keeping this small and avoiding surface area for regressions only the items listed in the description were migrated.